docs/src/pages/components/links/links.md

```diff
@@ -16,24 +16,24 @@ You can leverage its properties.
 However, the Link has different default properties than the Typography:
 
 - `color="primary"` as the link needs to stand out.
-- `variant="inherit"` as the link will, most of the time, be used as a child of a Typograpy component.
+- `variant="inherit"` as the link will, most of the time, be used as a child of the Typography component.
```

Comment:
I prefer `a Typography component` here.

Reply:
I changed "a" to "the" since it's the one `Typography` component. Also, elsewhere in this page it's referred to as `the Typography component`.

Reply:
@joshwooding So, which do we go with? "a" or "the"?

Reply:
@kyarik In the context of the sentence to me, it's referring to the instance of a Typography component, not the implementation. cc @mbrookes

Reply:
Maybe it would be better to just simplify the sentence to "`variant="inherit"` as the link will, most of the time, be used as a child of `Typography`?

Reply:
I agree with @joshwooding 

```suggestion
- `variant="inherit"` as the link will, most of the time, be used as a child of a Typography component.
```